ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -162,3 +162,16 @@ PMP_SS_VSC_NONCANONICAL_2:
   - mixing_valve_percentage_command
   - chilled_supply_water_temperature_setpoint
   - chilled_supply_water_temperature_sensor
+
+
+PMP_SS_REFPM_REFSM_OA_CPM2X:
```

Comment:
If this is an air-source heat pump it does not belong in the `PMP` file, this would be a `CH` or perhaps a new general type altogether.